ui/helpers/utils/util.js

```diff
@@ -366,6 +366,7 @@ const solidityTypes = () => {
     'bytes',
     'int',
     'uint',
+    'uint256',
```

Comment:
This should already be included in `uints` below?

Reply:
You are correct, I updated my PR.